index.html

```diff
@@ -320,5 +321,113 @@ <h3 class="pricing__item-title">Premium</h3>
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="9"
+                  height="16"
+                  viewBox="0 0 9 16"
+                >
+                  <g>
+                    <g>
+                      <path
+                        fill="#474747"
+                        d="M8.86 7.675L.815.122a.504.504 0 0 0-.683 0 .433.433 0 0 0 0 .641l7.704 7.23L.13 15.226a.432.432 0 0 0 0 .64.505.505 0 0 0 .34.135.488.488 0 0 0 .339-.135l8.047-7.553a.43.43 0 0 0 .004-.637z"
+                      />
+                    </g>
+                  </g>
+                </svg>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="news__item">
+          <img src="./image/DesignTheory.png" />
+          <div class="news__item-description">
+            <h3 class="news__item-title">Design Theory</h3>
+            <p class="news__item-authored">
+              Posted By: <span>Admin | 07 Sep,2018</span>
+            </p>
+            <p class="news__item-text">
+              Ut enim ad minim veniquis nostrud lieon tear exercitation
+              ullamco.If you use this site to donating a bandwidth bill.
+            </p>
+            <div class="news__item-link">
+              <p class="news__item-link-text">Read More</p>
+              <div class="news__item-link-svg">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="9"
+                  height="16"
+                  viewBox="0 0 9 16"
+                >
+                  <g>
+                    <g>
+                      <path
+                        fill="#474747"
+                        d="M8.86 7.675L.815.122a.504.504 0 0 0-.683 0 .433.433 0 0 0 0 .641l7.704 7.23L.13 15.226a.432.432 0 0 0 0 .64.505.505 0 0 0 .34.135.488.488 0 0 0 .339-.135l8.047-7.553a.43.43 0 0 0 .004-.637z"
+                      />
+                    </g>
+                  </g>
+                </svg>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="news__item">
+          <img src="./image/TechknologyIdeas.png" />
+          <div class="news__item-description">
+            <h3 class="news__item-title">Techknology & Ideas</h3>
+            <p class="news__item-authored">
+              Posted By: <span>Admin | 07 Sep,2018</span>
+            </p>
+            <p class="news__item-text">
+              Ut enim ad minim veniquis nostrud lieon tear exercitation
+              ullamco.If you use this site to donating a bandwidth bill.
+            </p>
+            <div class="news__item-link">
+              <p class="news__item-link-text">Read More</p>
+              <div class="news__item-link-svg">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="9"
+                  height="16"
+                  viewBox="0 0 9 16"
+                >
+                  <g>
+                    <g>
```

Comment:
remove `<g>
                    <g>`

and `</g>
                  </g>`

on svg.

`g` in svg like empty `div` you can delete this tags